R/utils.R

```diff
@@ -41,6 +41,18 @@ os_is_windows <- function() {
   isTRUE(.Platform$OS.type == "windows")
 }
 
+os_is_wsl <- function() {
+  # Avoid errors if checking cmdstan_path before it has been set
+  wsl_in_path <- tryCatch({
+      grepl("wsl-cmdstan", cmdstan_path())
```

Comment:
Should we maybe instead have an internal variable that is set on calls to `set_cmdstan_path()` instead? Just grepping the path is not a big deal so this is optional to avoid the tryCatch.